sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,121 @@
+      <Units>hourangle</Units>
       <Count>1</Count>
     </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection at corner splitted sensors.</Description>
     <item>
-      <EFDB_Name>visit</EFDB_Name>
-      <Description>Image visit number.</Description>
-      <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>directoryPath</EFDB_Name>
-      <Description>Absolute directory path to wavefront data collection.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>fieldRA</EFDB_Name>
-      <Description>Right ascension of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>decl</EFDB_Name>
+      <Description>Declination of the field in ICRS coordinates.</Description>
+      <IDL_Type>float</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>fieldDEC</EFDB_Name>
-      <Description>Declination of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>pa</EFDB_Name>
+      <Description>Sky position angle of the observation. As defined by the IAU, this the angle measured relative to the north celestial pole (NCP), turning positive into the direction of the right ascension.</Description>
+      <IDL_Type>float</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
+      <Description>Name of the filter used in the observation.</Description>
+      <IDL_Type>string</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>cameraRotation</EFDB_Name>
-      <Description>Camera rotation angle.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>userGain</EFDB_Name>
-      <Description>Gain value between 0 and 1. Set to -1 to ignore user gain.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>mode</EFDB_Name>
+      <Description>Enumeration specifying the wfs mode.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processIntraExtraWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection by camera piston.</Description>
+    <EFDB_Topic>MTAOS_command_preProcess</EFDB_Topic>
+    <Description>Pre-process image for WEP.</Description>
```

Comment:
I think we agreed to write down the details instead of just using the vague words of "Pre-process image".

Reply:
I think we will have to wait to write down the details for when we know what they are. At this point it is a bit early to add details here.

Reply:
I do not think so. Instead, for a problem, I think if we could ask the question clearly, we had solved the half of it already. I still insist a good design before the implementation is critical.

Reply:
Not sure I understand your point. We know we will have to do "pre-processing" to those images. What are all these processes? I don't know. I know it will involve ISR and probably some other tasks. We will only know when we are actually processing real data.

I don't think a good design is one that specify every detail beforehand, but one that is flexible enough to evolve with our understanding of the problem.

Reply:
Sure. Maybe this is the best option at this moment. Thanks!